apps/protocol-frontend-e2e/cypress.config.ts

```diff
@@ -1,22 +1,25 @@
 import { defineConfig } from 'cypress';
 
+
 const {Client} = require('pg');
 
-function queryDB(connectionInfo, query) {
+function queryDB(query) {
+  const connectionInfo = 'postgresql://postgres:test@localhost:5432/govrn'; //Cypress.env('DATABASE_URL')
```

Comment:
Why change this? Env would probably be preferred here.

Reply:
Currently addressed. My bad thanks for catching that.